packages/file_selector/file_selector_web/test/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: regular_integration_tests
+publish_to: none
+
+environment:
+  sdk: ">=2.2.2 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  file_selector_web:
+    path: ../
+  file_selector_platform_interface:
+    path: ../../file_selector_platform_interface
+  google_maps: ^3.4.4
+  flutter_driver:
+    sdk: flutter
+  flutter_test:
+    sdk: flutter
+  e2e: ^0.6.1
```

Comment:
Note: this is not called e2e anymore, it's now `integration_test`, and can (should) be loaded from the path. Take a look at the latest push I did for `google_maps_flutter_web` to "migrate" to the new package:

https://github.com/flutter/plugins/commit/a9ac52f3fe10034ef7f48a98f9112bca0666d3e9

Reply:
Why is `integration_test` also in `google_maps_flutter_web` pubspec.yaml instead of just in `google_maps_flutter_web/test`?

Reply:
The "publishable" check fails because it doesn't recognize google_maps_flutter_web/test as a separate package (currently, these tests are put inside an "example" directory, but most of the web packages don't have a stand-alone example)